app/services/braze_service.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class BrazeService
+  class << self
+    def send_swa_my_collection_email(submission_id)
+      submission = Submission.find(submission_id)
+      raise 'Still processing images.' unless submission.ready?
+
+      raise 'User lacks email.' if submission.email.blank?
+
+      recipient = [
+        {
+          external_user_id: submission.user.gravity_user_id,
+          trigger_properties: {
+            email_subject: '[TESTING] SWA Braze integration'
```

Comment:
This needs a final copy.

Reply:
We also need the template ready to be able to send other fields here.

Reply:
I think we can use the existing `swa_my_collection` feature flag if you would like to try triggering an email.